proxy.conf.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
Is there only reason for proxy that user context cookie?

Reply:
and file downloads, for now